### ISSUE TITLE
Messages Containing URL href Don't Link

### ISSUE THREAD

**Issue by dhanar10** [EllisV/hubot-skyweb#3]
Hi @EllisV,

I noticed that you apply escapeSendMessage function on every message sent. This makes it impossible to send clickable URL. What is the reason for this? As workaround, I disabled this function and my messages got send correctly and clickable URL works.

Test Case:
Send message: `<a href="http://www.google.com"> Google</a>`

Actual:
Message received: `<a href="http://www.google.com"> Google</a>`

Expected:
Message received: <a href="http://www.google.com"> Google</a> (the text color is blue and when clicked, it opens www.google.com)

**Comment by ellisv** [comment]
Hey @dhanar10,

This is a commit behind it: https://github.com/EllisV/hubot-skyweb/commit/39dcdf3752d4795740c975ee8602cc45c360e17f
A reason is that when sending `hello < world` message skype api would lead into an error (same for `&` and `>` chars at least). I wasn't aware that you are able to send a linked text via Skype. Is it possible to do so via desktop client?

**Comment by dhanar10** [comment]
>  Is it possible to do so via desktop client?

@EllisV I just realize that I was chatting with my bot using Skype Web - and without escapeSendMessage it was working correctly. Now, I am at work and try to chat with my bot using Skype Desktop on Windows and my bot keeps failing to send message.

I start to think that this issue is actually Skype Web issue. URL send by the bot don't link if it is received by Skype Web, but with Skype Windows Desktop the URL link is displayed correctly.

So in summary,

Test Case:
Send message: http://www.google.com

Actual:
In Skype Web - Message received: http://www.google.com (but it is a normal text, not hyperlink)
In Skype Desktop - Message received: http://www.google.com (it is a hyperlink - this is what we want)

Expected:
Message received: http://www.google.com (hyperlink in both Skype Web and Desktop)

What do you think?

**Comment by dhanar10** [comment]
@EllisV After looking at it a couple of weeks, I think this issue is not hubot-skyweb's issue. Therefore, I think it is better if I close it. Thanks for your response.
